CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing to a Rearc AWS Data Exchange product
+
+🎉🥳 First of all, **THANK YOU** for being interested in contributing to one of our projects 🎉🥳
+
+From the start of our AWS Data Exchange (ADX) initiative we knew we wanted to have an open dialogue with our subscribers. While we try our best to optimize the packaging of our ADX products to be as accommodating as possible, we appreciate there may be use-cases we have overlooked. By contributing to this project, you have an opportunity to improve the usability of the data included with this product not only for yourself, but for anyone who subscribes to this product through ADX. If your contribution is applicable to more than one of our ADX listings, we will gladly improve those other products as well!
```

Comment:
Overall I think this Markdown is a bit to get through. I'd consider this paragraph for cutting.

Reply:
Removed the paragraph that was on line 5